orchestrator-salt/src/main/resources/salt/salt/resize/scripts/resize-storage-volumes.j2

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+source /opt/salt/scripts/format-and-mount-common.sh
+
+LOG_FILE="/var/log/resize-storage-volumes.log"
```

Comment:
Add the logfile to the `vm-logs.json` so that the diagnostic tool can collect the logs